ci/ray_ci/bisect/release_test_bisect.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
this script is from the current pipeline definition: https://buildkite.com/ray-project/bisect/settings/steps